jenkins/values-agents.yaml

```diff
@@ -24,5 +24,5 @@ jenkins:
   additionalAgents:
     kanikoEight:
       podName: kaniko-precached-jdk8
-      customJenkinsLabels: "kaniko-precached-jdk8"
+      customJenkinsLabels: "kaniko-precached-jdk8 kaniko"
       tag: latest-jdk8
```

Comment:
BSA mentioned that we don't need JDK8 for DestSol anymore, and I believe we also no longer need it for Terasology.
@Cervator are you aware of any other part of our projects that might still need JDK8? Otherwise I believe, we don't need a JDK8 agent anymore.